digiwf-libs/digiwf-spring-cloudstream-utils/digiwf-spring-cloudstream-utils-core/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>digiwf-spring-cloudstream-utils</artifactId>
+        <groupId>io.muenchendigital.digiwf</groupId>
+        <version>0.10.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>digiwf-spring-cloudstream-utils-core</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-stream</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.muenchendigital.digiwf</groupId>
```

Comment:
@dominikhorn93 Is this dependency needed?

Reply:
Shouldnt it be api?

Reply:
Yes!